src/Tizen.WebView/Tizen.WebView/Settings.cs

```diff
@@ -31,6 +31,23 @@ internal Settings(IntPtr handle)
             _handle = handle;
         }
 
+        /// <summary>
+        /// Whether the Auto Fitting can be fit to width.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
```

Comment:
since_tizen should be 6 for master branch.
Plz refer to Branches section of README.md.
Plus, you have to do ACR process as well, in order to add new API.